lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -900,6 +903,7 @@ def test_static_tab_detail_get(self):
         self.assertTrue(tab_contents is not None)
         self.assertEqual(tab_contents, self.static_tab1.data)
 
+        # now get static tab by url_slug
```

Comment:
We should break this test up into two separate tests -- one to test retrieval by tab_name and one to retrieve by url_slug